src/generic/grid.cpp

```diff
@@ -5889,6 +5886,56 @@ void wxGrid::RefreshBlock(int topRow, int leftCol,
     }
 }
 
+void wxGrid::RefreshArea(int area, wxRect* rect, bool erasebg)
+{
+    if ( area == wxGridArea::All )
+    {
+        wxScrolledCanvas::Refresh(erasebg);
```

Comment:
Does this not take `rect` into account intentionally? If so, there should be a comment here explaining why as this doesn't seem obvious.